libcontainer/cgroups/devices/devicefilter.go

```diff
@@ -23,14 +22,14 @@ const (
 	license = "Apache"
 )
 
-// DeviceFilter returns eBPF device filter program and its license string
-func DeviceFilter(rules []*devices.Rule) (asm.Instructions, string, error) {
+// deviceFilter returns eBPF device filter program and its license string.
+func deviceFilter(rules []*devices.Rule) (asm.Instructions, string, error) {
```

Comment:
Probably no need to unexpose `DeviceFilter`, `Emulator`, etc.
Or this could be another PR.

Reply:
It's a separate commit so I can just drop it.

Or, if moving everything in a single package is not that good (let's see what @cyphar says), this can be undone, too.

Reply:
Moving it to a single package is fine. Honestly I probably should've kept it in this package in the first place (I wish Go had project-level internal symbols that didn't require you to restructure everything to use `internal/`).